app/models/portfolio_item.rb

```diff
@@ -18,8 +18,15 @@ class PortfolioItem < ApplicationRecord
   attribute :metadata, ActiveRecord::Type::Json.new
 
   def metadata
-    user = UserContext.new(Insights::API::Common::Request.current!, nil)
+    {:user_capabilities => user_capabilities}
+  end
+
+  private
+
+  def user_capabilities
```

Comment:
@eclarizio Can this shared code be moved into a higher level object maybe application_record and we can control it by calling metadata from Portfolio and PortfolioItem models. @hsong-rh is doing something similar in PR https://github.com/RedHatInsights/approval-api/pull/313